lib/kubernetes-deploy/kubectl.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module KubernetesDeploy
+  module Kubectl
+    def self.run_kubectl(*args, namespace:, context:, log_failure: true)
+      args = args.unshift("kubectl")
+      args.push("--namespace=#{namespace}") unless namespace.blank?
```

Comment:
It's up to you, but `if namespace.present?` may be slightly better readable than `unless namespace.blank?`